custom/icds_reports/static/js/directives/ls-launched/ls-launched.directive.js

```diff
@@ -0,0 +1,97 @@
+/* global d3 */
+var url = hqImport('hqwebapp/js/initial_page_data').reverse;
+
+function LSLaunchedController($scope, $routeParams, $location, $filter, icdsCasReachService, locationsService,
+    dateHelperService, navigationService, userLocationId, storageService, haveAccessToAllLocations,
+    baseControllersService, isAlertActive, isMobile, haveAccessToFeatures) {
+    baseControllersService.BaseController.call(this, $scope, $routeParams, $location, locationsService,
+        dateHelperService, navigationService, userLocationId, storageService, haveAccessToAllLocations,
+        haveAccessToFeatures, isMobile);
+    var vm = this;
+    vm.isAlertActive = isAlertActive;
+    vm.usePercentage = false;
+    vm.serviceDataFunction = icdsCasReachService.getLSLaunchedData;
+
+    vm.label = "LSs Launched";
+    vm.steps = vm.getSteps('/icds_cas_reach/ls_launched/');
+    vm.data = {
+        legendTitle: 'Total number of Lady supervisors who have ever submitted any form in the LS app',
+    };
+    vm.rightLegend = {
+        info: 'Total number of Lady supervisors who have ever submitted any form in the LS app',
+    };
+    vm.filters = ['age', 'gender'];
```

Comment:
where is this age and gender filter used on ls launched page?

Reply:
Actually this filters array is used to get the filters that are not to be displayed on a particular report. (Kind of counter-intuitive, just following the pattern)